docs/docs/releases/0.2.0.md

```diff
@@ -1,302 +1,550 @@
-- Improvements to installation from scratch documentation _[#3945](https://github.com/mathesar-foundation/mathesar/pull/3945 "Docs improvements for build from scratch")_
+### New User Guide
+
+We've overhauled our [User Guide](../user-guide/index.md) to document all new features and more thoroughly explain the inner-workings of Mathesar. You might even learn a thing or two about PostgreSQL by reading it!
+
+[#3910](https://github.com/mathesar-foundation/mathesar/pull/3910 "Permissions Documentation Updates")
+[#3945](https://github.com/mathesar-foundation/mathesar/pull/3945 "Docs improvements for build from scratch")
+[#4031](https://github.com/mathesar-foundation/mathesar/pull/4031 "Comprehensive documentation improvements")
+
+### More in-app help
+
+You'll also find more help within the application itself. Look for small gray help bubbles to explain concepts and provides links to more detailed information within the User Guide.
+
+<!-- TODO_SCREENSHOT: Library management "Items" table page, table inspector open, all sections collapsed except for "Relationships". Animated Gif showing cursor hovering over the three help bubbles within this table inspector section, giving the viewer time to quickly read each help bubble -->
+
+[#4108](https://github.com/mathesar-foundation/mathesar/pull/4108 "Add more in-app help bubbles")
+[#4135](https://github.com/mathesar-foundation/mathesar/pull/4135 "Resources section and donate in-app")
+[#4144](https://github.com/mathesar-foundation/mathesar/pull/4144 "Fix regression in help bubble placement")
+[#4121](https://github.com/mathesar-foundation/mathesar/pull/4121 "Update public schema notice")
+[#4098](https://github.com/mathesar-foundation/mathesar/pull/4098 "Fix regression: Display the Welcome page when there are no databases")
+
+### Multi-versioned documentation
+
+Our documentation site https://docs.mathesar.org/ now features documentation for multiple Mathesar versions. See the version-switcher within the top navigation bar to read documentation for older versions of Mathesar.
+
+[#3920](https://github.com/mathesar-foundation/mathesar/pull/3920 "Multi-versioned docs — `0.1.7` updates")
+[#3967](https://github.com/mathesar-foundation/mathesar/pull/3967 "Document multi-version docs system")
+
+
+## Features removed
+
+### Sharable tables and explorations are no longer available
+
+Back in version 0.1.3, we added support for "sharing" tables and explorations via a link that would allow anonymous users to view data. In Mathesar 0.2.0, we've removed this feature in order to speed up the development of our new PostgreSQL-based access control system. We hope to add this sharing feature back in a future release, but it will require some more design and thought in order to be compatible with the new access control system. If you were relying on this feature, we'd like to hear from you! Please [open an issue](https://github.com/mathesar-foundation/mathesar/issues/new?template=feature_request.md) to let us know.
```

Comment:
I don't think the "hope to add back in" is needed. Just skip to the "If you were relying ..." part.

Reply:
Addressed via c13698d7b62e91826560728acc31a6b80102bf95